src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/RightCurlyCheck.java

```diff
@@ -444,6 +491,20 @@ private static boolean hasLineBreakBefore(DetailAST rightCurly) {
     }
 
     /**
+     * Checks if right curly is alone on line in token the lambda.
+     * @param details for validation.
+     * @return true, if right curly is alone on line.
+     */
+    private static boolean isAloneInLambda(Details details) {
```

Comment:
Method has no `lambda` specific code. Lets make it generic method so we can use it for anything in the future.